src/hcrystalball/feature_extraction/_holiday_transformer.py

```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import logging
```

Comment:
I guess we want here following pattern, correct?
`import logging
logger = logging.getLogger(__name__)
...
logger.warning("...")`

Reply:
@pavelkrizek I see no logging used here - can you remove that import please?